lib/sphero.js

```diff
@@ -24,6 +24,23 @@ function classCallCheck(instance, Constructor) {
   }
 }
 
+/**
+ * Creates a new sphero instance
+ *
+ * You can pass the following options:
+ *
+ * opts:
+ *   adaptor: {String} [optional] (bluetooth|serial) default "serial"
+ *   sop2: {number} [optional] default 0xFD
+ *   timeout: {number} [optional] (deadtime in miliseconds) default 500
+ *
```

Comment:
These can just be documented as further `@param` tags, e.g. `@param {Object} [opts.adaptor] an adaptor object for Sphero communication`